ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -197,7 +197,7 @@ QtObject {
     }
 
     function getNameForAddress(address) {
-        let name = getNameForWalletAddress(address)
+        var name = getNameForWalletAddress(address)
```

Comment:
huh, shouldn't the compiler have complained about this?

Reply:
We might have had some js warnings and side effects we didn't notice

Reply:
oohhhh this is QML, not Nim. Disregard my comment!

Reply:
This was fine defining a variable in the scope instead of my change to global, I will revert it. Thanks for the heads up, I get confused with all the languages flying around :)